### PR TITLE
Reintroduce the isLast variable

### LINKED ISSUE #141: Runtime Error due to missing variable
**Describe the bug**
Runtime error occurs while running the repo locally due to a missing variable (`isLast`). Deployment failure due to the same bug - https://github.com/saving-satoshi/saving-satoshi/commits/master 

**To Reproduce**

1. fetch the latest code from the master branch
2. `npm run dev`
3. go to `localhost:3000/chapters/chapter-1`
4. See the error

**Screenshots and/or logs**
Screenshot 2023-02-08 at 1 59 58 PM

**Additional context**
https://github.com/saving-satoshi/saving-satoshi/pull/140#discussion_r1099830161

### PR DESCRIPTION
**Pull request by shaavan**
fixes #141 

- This PR reintroduces the isLast variable to keep track of the last challenge and to fix the `Reference Error: Can't find variable: isLast"`

### CODE REVIEW

**Comment by ecurrencyhodler** [comment]
Thanks for the quick fix! @vinayaksh42 Can you review?

File: components/Navbar/Tab.tsx [hunk 1]
```diff
@@ -32,6 +32,7 @@ export default function Tab({
 
   const challengeId = isRouteLesson ? pathData.pop().split('-')[0] : undefined
   const isActive = challenge.lessonId === challengeId
+  const isLast = index == count - 1
```
Comment (afterburn) [inline components/Navbar/Tab.tsx:35]: Let's use `===` to make sure JS doesn't make any assumptions for us. Should work like this, but its a good practice to use `===` whenever you can.

**Review by afterburn — CHANGES_REQUESTED** [review]
Left a comment for a small improvement

**Comment by vinayaksh42** [comment]
> Thanks for the quick fix! @vinayaksh42 Can you review?

LGTM
Let's implement Kevin's suggestion for added improvement

**Review by vinayaksh42 — COMMENTED** [review]
LGTM
Let's implement Kevin's suggestion for added improvement

File: components/Navbar/Tab.tsx (hunk 1 above)
Comment (shaavan) [reply components/Navbar/Tab.tsx:35]: Sure! Let me fix this. Always avoid assumptions! ✌️

**Comment by shaavan** [comment]
Updated from ffab86197be36b02ad59289e7dd4262903881ebd to 3ba01cc2905cfe4bc61088ce74e9cd36fbe667a7 ([pr142.01](https://github.com/shaavan/saving-satoshi/commits/pr142.01) -> [pr142.02](https://github.com/shaavan/saving-satoshi/commits/pr142.02), [diff](https://github.com/shaavan/saving-satoshi/compare/pr142.01..pr142.02))
Addressed @afterburn comment.

Change:

- Replace `==` with `===`.

**Review by afterburn — APPROVED** [review]

**Pull request merged**